readme.md

```diff
@@ -53,3 +83,8 @@ brew install carthage
 cd $PROJECT_DIR
 carthage bootstrap
 ```
+
+<style>
```

Comment:
GitHub-flavored Markdown doesn’t support `<style>` tags, so it just shows up as gibberish at the bottom.

Reply:
Renders as `display:none` for me

Reply:
On GitHub?

![github](https://user-images.githubusercontent.com/1231218/58742948-dc630a00-83dd-11e9-9887-a470fb6aac2b.png)